src/main/java/salesforce/utils/Urls.java

```diff
@@ -0,0 +1,28 @@
+package salesforce.utils;
+
+import core.config.EnvConfig;
+
+/**
+ * This class has urls salesforce.
+ */
+public enum Urls {
+    PATH_LOGIN("https://login.salesforce.com/"),
+    FEATURE_URL_LIGHTNING("lightning/o/%s/list?filterName=Recent"),
+    WORKTYPE_CLASSIC(EnvConfig.getInstance().getBaseUrl() + "08q/o"),
```

Comment:
Probably this url is going to change, is there another way to define the worktype URL?